homeassistant/components/rabbitair/entity.py

```diff
@@ -0,0 +1,69 @@
+"""A base class for Rabbit Air entities."""
+from __future__ import annotations
+
+import asyncio
+import logging
+
+from rabbitair import Client, Model
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+MODELS = {Model.BioGS: "BioGS 2.0", Model.MinusA2: "MinusA2", Model.A3: "A3"}
+
+
+class RabbitAirBaseEntity(CoordinatorEntity):
+    """Base class for Rabbit Air entity."""
+
+    def __init__(
+        self, coordinator: DataUpdateCoordinator, client: Client, entry: ConfigEntry
+    ) -> None:
+        """Initialize the entity."""
+        super().__init__(coordinator)
+        self._client = client
+        self._entry = entry
+        # Use assert to suppress the type warning. We always set a unique ID in
+        # the config flow, so it should never be None.
+        assert entry.unique_id is not None
```

Comment:
Replaced it with `# type: ignore` annotation, just to avoid using `assert`. Although, in my opinion, this solution is worse, because it hides possible errors in our code and, therefore, is less safe.